marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/MergeDatabaseConfigTest.java

```diff
@@ -0,0 +1,89 @@
+package com.marklogic.hub.central.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MergeDatabaseConfigTest {
+
+    @Test
+    public void testMergeIndexConfigs() throws JsonProcessingException {
+        String entityServicesRangeIndex = "testEntityServicesRangeIndexForDHFPROD4704";
+        String entityServicesPathIndex = "testEntityServicesPathIndexForDHFPROD4704";
+        String databaseRangeIndex = "testRangeIndexForDHFPROD4704";
+        String databasePathIndex = "testPathIndexForDHFPROD4704";
+        String databaseField1 = "language";
+        String databaseField2 = "forest";
+
+        String indexConfig = "{\n" +
```

Comment:
For future reference - ml-app-deployer has a bunch of POJOs to build this stuff, which is a lot easier to write and read than doing inline JSON in Java. See UpgradeProjectTest for examples of using ElementIndex and Database.

Reply:
Awesome, updated the test to use the POJOS.